admin_guide/overcommit.adoc

```diff
@@ -112,6 +112,26 @@ system tasks or daemons need more memory than was accounted for in the node's
 resource reservation. If a container specifies a limit on memory, it is
 immediately killed if it exceeds the limit amount.
 
+[[overcommit-ephemeral-storage]]
+=== Ephemeral Storage
```

Comment:
Need to provide an explanation of ephemeral storage. We need to explain to the reader what ephemeral storage is, how it behaves, what types of ephemeral storage are available, what components (like containers) can use this type of storage, etc.

Reply:
Please see `architecture/additional_concepts/ephemeral-storage.adoc`; that may answer the questions.  I'd rather put the overall explanation in one place and provide a link to it rather than explaining everywhere what ephemeral storage is.

Reply:
Sure, linking to the information is fine, but I still think it would be helpful to include one or two sentences stating what ephemeral storage is. Maybe something like "Ephemeral storage is non-persistent storage that is destroyed when the pod is restarted."

Reply:
OK